src/rpc/rpc-handler.ts

```
@@ -90,11 +90,22 @@ export class RPCHandler {
    * @param   {Function} callback Will be invoked with the returned result or if the rpc failed
    *                              receives to arguments: error or null and the result
    */
-  public make (name: string, data: any, callback: (error: string, data: any) => void): Promise<any> | undefined {
+  public make (name: string, data: any, callback?: RPCMakeCallback): Promise<any> | undefined {
     if (typeof name !== 'string' || name.length === 0) {
       throw new Error('invalid argument name')
     }
 
+    const cb = callback && typeof callback === 'function'
```

If callback is provided and is not a function should error, not fail silently